webservice/webservice.py

```diff
@@ -402,6 +404,28 @@ def export_to_firecloud():
     fc_lambda_protocol = os.getenv("FC_LAMBDA_PROTOCOL", "https")
     fc_lambda_domain = os.getenv("FC_LAMBDA_DOMAIN", domain)
     fc_lambda_port = os.getenv("FC_LAMBDA_PORT", '443')
+
```

Comment:
Please create a new endpoint. Do not break the behavior of the current endpoint.